ospd/ospd.py

```diff
@@ -1174,15 +1174,32 @@ def run(self) -> None:
         """ Starts the Daemon, handling commands until interrupted.
         """
 
+        self.scan_collection.data_manager = multiprocessing.Manager()
```

Comment:
I really don't think this line belongs into the daemon class. I would call the data_manager an internal thing of the scan_collection which should not be accessed by the outside. Either create the manager when the scan_collection is created (constructor) or add some init method which is called later.